frontend/src/entry/overview/Overview.tsx

```diff
@@ -13,18 +14,24 @@ type OverviewPageKeys = (typeof OverviewPage)[keyof typeof OverviewPage]
 
 type OverviewProps = {
   entry: EntryInterface
+  readOnly?: boolean
 }
 
-export default function Overview({ entry }: OverviewProps) {
+export default function Overview({ entry, readOnly = false }: OverviewProps) {
   const page: OverviewPageKeys = useMemo(
     () => (entry.card && entry.card.schemaId ? OverviewPage.FORM : OverviewPage.TEMPLATE),
     [entry.card],
   )
 
-  return (
+  return readOnly ? (
+    <MessageAlert
+      severity='warning'
+      message='This mirrored model has no model card. Please export the model card from the source model.'
```

Comment:
If I manually a model card data into the DB then I still see this message rather than the model card?